packages/webapp/pages/onboarding.tsx

```diff
@@ -394,12 +397,7 @@ export function OnboardPage(): ReactElement {
   }
 
   if (finishedOnboarding) {
-    return (
```

Comment:
Did we ever use this return?

Reply:
Resolved now

Reply:
The old one? It should be part of the new component already.